addons/sourcemod/scripting/scp_sf.sp

```diff
@@ -3400,7 +3403,17 @@ public Action SendProp_OnAliveMulti(int entity, const char[] propname, int &valu
 	return Plugin_Changed;
 }
 
-public Action SendProp_OnTeam(int entity, const char[] propname, int &value, int client) 
+public Action SendProp_OnTeamMulti(int entity, const char[] propname, int &value, int client, int target)
+{
+	return OnTeam_Internal(client, value);
+}
+
+public Action SendProp_OnTeam(int entity, const char[] propname, int &value, int client)
+{
+	return OnTeam_Internal(client, value);
+}
```

Comment:
Instead of needing to create function for each versions, can just reuse with `#if` on just function params, like this
```
#if defined SENDPROXY_LIB
public Action SendProp_OnTeam(int entity, const char[] propname, int &value, int client, int target)
#else
public Action SendProp_OnTeam(int entity, const char[] propname, int &value, int client)
#endif
{
	return OnTeam_Internal(client, value);
}
```

Same to `SendProp_OnClass` and `SendProp_OnClientClass`